src/Api/Api.js

```diff
@@ -11,6 +11,16 @@ export const getArticles = async () => {
       return response;
     } catch (error) {
```

Comment:
You probably don't need to catch the error in here, nor should you need to throw your own - when entering a catch block, the error has already been generated for you. You could simply catch in the component and update UI accordingly